Activity_Selection/Activity_Selection.dart

```diff
@@ -0,0 +1,38 @@
+/* The Activity Selection Problem
+You are given n activities with their start and finish times. 
+Select the maximum number of activities that can be performed by a single person, 
+assuming that a person can only work on a single activity at a time. 
+*/
+class ActivitySelection 
+{
+    void activities(List<int> start, List<int> finish, int n) 
```

Comment:
The indentation of this function is more than 4 spaces. You can make this out by seeing it. 
Please fix this function. Rest is fine.

Reply:
![image](https://user-images.githubusercontent.com/37381075/57144195-2e644000-6dde-11e9-99f2-e20976b82929.png)
The editor says it's fine.  I have added the blank line at the end.

Reply:
Hi, I know it shows correct in your editor but it has different indentation from the rest of file here. 
Can you do one thing? 
Just manually delete the spaces infront of each line and add required no. of spaces : 4 per block and then commit.
I think then it should work. Most probably, tabs and spaces would have got mixed.

Reply:
> 
> 
> Hi, I know it shows correct in your editor but it has different indentation from the rest of file here.
> Can you do one thing?
> Just manually delete the spaces infront of each line and add required no. of spaces : 4 per block and then commit.
> I think then it should work. Most probably, tabs and spaces would have got mixed.
Okay, on it.

Reply:
I've made another commit. Please check

Reply:
In the editor it is just as you want it to be. Even if I manually do 4 spaces it is the same thing as the commit I've already made. And the screenshot above shows it. On the github viewer however what you're saying is correct. The brackets are not in line. When I do the brackets again it reverts back to the original commit so basically no changes are made. Is there any other way? I have tried both tabs and spaces and both of them give the same result.